modules/thunder_paragraphs/js/paragraphs.click_edit.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * @file
+ * Provides click-to-edit functionality for paragraphs.
+ */
+
+(function paragraphsClickEditModule(Drupal, once) {
+  const PARAGRAPH_SELECTOR =
+    '.paragraphs-item, .paragraph-form-item--has-preview, [id^="field-paragraphs-"][id*="-item-wrapper"]';
```

Comment:
We could make this configurable via context settings to make it more easy for people to provide their own selector
Nitpic: Use camelCase for names in js